app/templates/drupal/octane/8.x/Gruntconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "srcPaths": {
+    "drupal": "src"
+  },
+  "domain": "project.vm",
+  "packages": {
+    "srcFiles": ["!sites/*/files/**", "!xmlrpc.php", "!modules/php/*"],
+    "projFiles": ["README*", "bin/**", "hooks/**", "src/*.make", "vendor/**", "composer.*"],
+    "dest": {
+      "docroot": "build/html",
+      "devResources": ""
+    }
+  },
+  "phpcs": {
+    "path": "vendor/bin/phpcs"
+  },
+  "phpmd": {
+    "path": "vendor/bin/phpmd"
+  },
+  "drush": {
+    "cmd": "vendor/bin/drush"
+  },
+  "behat": {
+    "flags": "--tags ~@wip"
+  },
+  "eslint": true,
+  "scripts": {
+    "update": "<%= config.drush.cmd %> <%= config.alias %> features-revert-all -yv"
```

Comment:
Yep, just copied it, so we can have a post 1.0 issue to improve this across the distros.